src/setup/demo.md

```diff
@@ -0,0 +1,352 @@
+
+# Class Metrics
+class MyMetrics(tools.Metrics):
+
+# Class Algo
+class MyAlgo(tools.algo.Algo):
+```
+
+This section is documented [here](https://github.com/SubstraFoundation/substra-tools/blob/master/docs/api.md).
+
+### Edit your hosts file
+
+As the demo instance is a demo instance, you will have to create a link between the server ip address (that we provided you together with your credentials) and the 2 nodes domains used in the demo:
+
+```sh
+# Replace <CHANGE_ME> with the ip that were provided together with your credentials
+echo "<CHANGE_ME> substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com" | sudo tee -a /etc/hosts
+```
+
+This will add a line like the following to the file `/etc/hosts` of your machine:
+
+```sh
+# 001.002.003.004     substra-backend.node-1.com substra-backend.node-2.com substra-frontend.node-1.com substra-frontend.node-2.com
+```
+
+In order to check that your local substra client can reach the demo server, you can use the following command:
+
+```sh
+# Quick test on the backend of the node-1
+curl substra-backend.node-1.com/readiness
+```
+
+You should receive an `"OK"` answer, if not please check again your `/etc/hosts` file. You can also use your browser to visit [substra-frontend.node-1.com](http:substra-frontend.node-1.com) & [substra-frontend.node-2.com](http:substra-frontend.node-2.com) (without https, because this is demo).
+
+### Login
+
+You can now configure your local Substra client, *with the credentials that were provided to access the demo instance*, to connect to the `node-1` which hosts the organisation `org-1`:
+
+```sh
+# Configure the client, with a profile, a user, a password and an url
+substra config --profile <PROFILE> -u <USER> -p "<PASSWORD>" http://substra-backend.node-1.com
+
+# Login with the profile you created (located in your home folder: ~/.substra)
+substra login --profile <PROFILE>
+
+# You can then list the network nodes that are on the Substra network
+substra list node --profile <PROFILE>
+
+# And list, for example, the registered datasets in the node you are logged in
+substra list dataset --profile <PROFILE>
+```
+
+## 2. Follow Titanic example
+
+With this example you will find everything you need to run a first end-to-end example with the tabular data of the Titanic: see this [repository](https://github.com/SubstraFoundation/substra/blob/master/examples/titanic/README.md) for further details.
+
+If you feel more comfortable working with local files, you can clone the [Substra repository](https://github.com/SubstraFoundation/substra) to get the full examples located in the `/example` folder.
+
+Please note that each resource is referenced by its hash in Substra, a unique identifier, so if you want to try out the ready-to-use Titanic example, you will need to edit, at least, the python files and the `description.md` for the assets (like adding a `print()` statement and editing the `description.md` files).
```

Comment:
Is it working if everybody add a "print()"? A random name is not better?

Reply:
In fact I was thinking about this section, but you are right. But printing something won't break anything